core/base-service/transform-example.js

```diff
@@ -46,66 +43,45 @@ function validateExample(example, index, ServiceClass) {
     `Example for ${ServiceClass.name} at index ${index}`
```

Comment:
This validation will fail if there's any extra keys, such as `previewUrl`, which is what now ensures there's no previewUrl right?

Reply:
Correct, exactly.